interpreters/duktape/Makefile

```diff
@@ -27,6 +27,8 @@ DUKTAPE_URL_BASE = https://github.com/svaarala/duktape/releases/download/
 DUKTAPE_URL      = $(DUKTAPE_URL_BASE)v$(DUKTAPE_VERSION)/$(DUKTAPE_TARBALL)
 
 CSRCS = $(DUKTAPE_UNPACK)/src-noline/duktape.c
+CSRCS += $(DUKTAPE_UNPACK)/extras/console/duk_console.c
+CSRCS += $(DUKTAPE_UNPACK)/extras/print-alert/duk_print_alert.c
```

Comment:
I asked a similar question in a merged PR, don't you need to extend clean to delete the object files generated?
With how the makefile is setup the objects are generated in `$(DUKTAPE_UNPACK)/extras/console/` for the `duk_console.c` as an example, right?

Reply:
I got it